### PR TITLE
Add Dockerfile for Node 16

### PR DESCRIPTION
**Pull request by MonkeyDo**
Jumped directly to the version 16.1.0, which becomes active LTS this October.

### CODE REVIEW

File: 16/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,77 @@
+FROM metabrainz/consul-template-base:v0.19.4
```
Comment (amCap1712) [inline 16/Dockerfile:1]: We use 0.18.5 as consul base in python brainz so might want to check this with zas.

Comment (amCap1712) [reply 16/Dockerfile:1]: Oh! I see this is our own image. Nevermind then.

**Review by paramsingh — APPROVED** [review]
lgtm if it builds locally

File: 16/Dockerfile (hunk 1 above)
Comment (MonkeyDo) [reply 16/Dockerfile:1]: For posterity: I looked to see if there is a more recent stable image with a proper tag (not `latest`), but didn't see any, so left it as before.

**Pull request merged**